lib/db/mem.js

```diff
@@ -363,6 +365,53 @@ module.exports = function (log, error) {
     return P.resolve({})
   }
 
+  Memory.prototype.verifyTokenCode = function (tokenVerificationCode, accountData) {
+    const uid = accountData.uid.toString('hex')
+    tokenVerificationCode = tokenVerificationCode.toString('hex')
+    let expired = false
+
+    const tokenCount = Object.keys(unverifiedTokens).reduce((count, tokenId) => {
+      const t = unverifiedTokens[tokenId]
+
+      if (t.uid.toString('hex') !== uid) {
+        return count
+      }
+
+      if (! t.tokenVerificationCodeHash || ! t.tokenVerificationCodeExpiresAt) {
+        return count
+      }
+
+      // Is code expired?
+      if (t.tokenVerificationCodeHash.toString('hex') === tokenVerificationCode) {
```

Comment:
Likewise here, it looks slightly off the compare a field named `tokenVerificationCodeHash` to a variable named `tokenVerificationCode`, we should either do the hashing or rename the field.